codebuild/common-linux.sh

```diff
@@ -27,7 +27,7 @@ sudo apt-get install libssl-dev
 
 cd ../
 
-mkdir install
+mkdir -p install
```

Comment:
Isn't -p just make all the directories required to satisfy the path argument? Why -p here?

Reply:
In the case of 32bit builds the libcrypto  build already created the directory.

-p basically makes it not error that the directory already exists.